doc/classes/JSONRPC.xml

```diff
@@ -15,6 +17,9 @@
 			<argument index="1" name="params" type="Variant">
 			</argument>
 			<description>
+				Returns a dictionary in the form of a JSON-RPC notification. Notifications are one-shot messages which do not expect a response.
+				- [code]method[/code] Name of the method being called.
+				- [code]params[/code] An array or dictionary of parameters being passed to the method.
```

Comment:
I would add a colon after the name of the key:
```suggestion
				- [code]method[/code]: Name of the method being called.
				- [code]params[/code]: An array or dictionary of parameters being passed to the method.
```

Same for all other descriptions with details on the keys.